pages/dashboard/projects.vue

```diff
@@ -1,22 +1,740 @@
+              :disabled="editLinks.discord.clear"
+              type="url"
+              maxlength="2048"
+              placeholder="Enter a valid URL"
+            />
+            <button
+              class="square-button label-button"
+              :data-active="editLinks.discord.clear"
+              @click="editLinks.discord.clear = !editLinks.discord.clear"
+            >
+              <TrashIcon />
+            </button>
+          </label>
+        </section>
+        <p>Changes will be applied to the following projects:</p>
+        <ul>
+          <li
+            v-for="project in selectedProjects.filter(
+              (it) => (it.permissions & EDIT_DETAILS) !== EDIT_DETAILS
+            )"
+            :key="project.id"
+          >
+            {{ project.title }}
+          </li>
+        </ul>
+        <p>
+          If a project you have selected is not in this list - you do not have
+          permission to edit it's details.
+        </p>
+        <div class="button-group">
+          <button class="iconified-button" @click="$refs.editLinksModal.hide()">
+            <CrossIcon />
+            Cancel
+          </button>
+          <button class="iconified-button" @click="bulkEditLinks()">
+            Apply
+          </button>
+        </div>
+      </div>
+    </Modal>
+    <ModalConfirm
+      ref="deleteBulkModal"
+      :title="`Delete Project${selectedProjects.length > 1 ? 's' : ''}`"
+      :description="`Deleting ${selectedProjects.length} project${
+        selectedProjects.length > 1 ? 's' : ''
+      }. This action is irreversable. Are you sure you want to continue? Only projects that you are owner of will be deleted.`"
+      :has-to-type="true"
+      :confirmation-text="$auth.user.username"
+      @proceed="bulkDeleteSelected()"
+    />
+    <ModalCreation ref="modal_creation" />
     <section class="universal-card">
-      <h2>Projects</h2>
+      <div class="title-button-group input-group">
+        <h2>Projects</h2>
+        <button
+          class="iconified-button brand-button"
+          @click="$refs.modal_creation.show()"
+        >
+          <PlusIcon />
+          Create a project
+        </button>
+      </div>
+
+      <div class="metrics">
+        <div class="metric">
+          <div class="label">Total Projects</div>
+          <div class="value">
+            {{ projects.length }}
+          </div>
+        </div>
+        <div class="metric">
+          <div class="label">Versions Released</div>
+          <div class="value">
+            {{ versions.length }}
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="universal-card">
+      <h2>Management</h2>
+      <p>You can bulk edit projects by selecting them on the table.</p>
+      <div class="button-group">
+        <button
+          class="iconified-button"
+          :disabled="selectedProjects.length === 0"
+          @click="$refs.editLinksModal.show()"
+        >
+          Edit Links
```

Comment:
```suggestion
          Edit links
```
Sentence case, not title case